Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift

```diff
@@ -50,25 +50,37 @@ extension AirRobeEndpoint {
         department: String? = nil,
         itemCount: Int? = nil
     ) -> AirRobeEndpoint {
+        var properties: [String: Any] = [
+            "source": AirRobeWidgetInfo.platform,
+            "version": AirRobeWidgetInfo.version,
+            "split_test_variant": UserDefaults.standard.SplitTestVariant?.splitTestVariant ?? "default",
+            "page_name": pageName,
+        ]
+        if let brand = brand {
+            properties["brand"] = brand
+        }
+        if let material = material {
+            properties["material"] = material
+        }
+        if let category = category {
+            properties["category"] = category
+        }
+        if let department = department {
+            properties["department"] = department
+        }
+        if let itemCount = itemCount {
+            properties["itemCount"] = itemCount
+        }
         let requestBody: [String: Any] = [
             "app_id": configuration?.appId ?? "",
             "anonymous_id": UIDevice.current.identifierForVendor?.uuidString ?? "",
             "session_id": sessionId,
             "event_name": eventName,
-            "properties": [
-                "source": AirRobeWidgetInfo.platform,
-                "version": AirRobeWidgetInfo.version,
-                "split_test_variant": UserDefaults.standard.SplitTestVariant?.splitTestVariant ?? "default",
-                "page_name": pageName,
-                "brand": brand,
-                "material": material,
-                "category": category,
-                "department": department,
-                "itemCount": itemCount
-            ]
+            "properties": properties,
+            "created_at": Date().iso8601withFractionalSeconds
         ]
         return AirRobeEndpoint(
```

Comment:
I'm not sure how it works in Swift but in widgets we needed to explicitly add this header: `"Content-Type": "application/json"`. This is a requirement for the API Gateway configuration we have.

Reply:
I added `application/json` to all the headers of the API call actually. You can find it